src/ScrollView/scrollView.scss

```diff
@@ -0,0 +1,113 @@
+@import '~gemini-scrollbar/gemini-scrollbar.css';
+@import '~sass-flex-mixins/src/sass-flex-mixins.scss';
+
+$scrollbarSize: 7px !default;
+$scrollbarSizeBig: 9px !default;
```

Comment:
I think we don't need but it should be more like `$scrollbar-size * 1.20`, that is valid sass but would leave decimals